server/routes.js

```diff
@@ -112,12 +112,29 @@ module.exports = [{
     }
   }
 }, {
-  // static routes using dist/, yay grunt
   method: 'GET',
-  path: '/assets/{param*}',
+  path: '/images/{param*}',
   handler: {
     directory: {
-      path: path.join(__dirname, '..', config.get('server.staticPath')),
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'images'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/scripts/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'scripts'),
+      listing: config.get('server.staticDirListing')
+    }
+  }
+}, {
+  method: 'GET',
+  path: '/styles/{param*}',
+  handler: {
+    directory: {
+      path: path.join(__dirname, '..', config.get('server.staticPath'), 'styles'),
```

Comment:
OK, I'll play around a bit more with this. I think I tried something similar when we were building the loads front end w/ Hapi+Angular and we saw some issues w/ 404 routing and a generic `/{param*}` route. But my memory is a bit hazy on the details &mdash; so I'll try and build a smaller test case and verify.